### ISSUE TITLE
managing import for different packages

### ISSUE THREAD

**Issue by savitaashture** [knative/client#208]
Currently in knative client we just have two separation for import
**Current:**
1. Standard 
2. Combined of Third-party packages,outside repo and within repo 
    ex:
```
	"flag"                                                                              // Standard

	"github.com/knative/client/pkg/kn/commands"              // within client and within knative
	homedir "github.com/mitchellh/go-homedir"                  // third-party
	""k8s.io/client-go/plugin/pkg/client/auth/gcp""                // third-party
        "github.com/knative/serving/pkg/apis/serving"              //  outside client and within knative
```
**Expected:**
Instead of combining all together we can separate  import based on there visibility like

1. Standard
2. Third-party packages
3. Within knative and outside client repo (e.g. github.com/knative/serving/pkg/apis/serving)
4. Within knative and client repo

Feedback is greatly appreciated.

**Comment by rhuss** [comment]
We are now using goimport for arranging the imports, so this is the "standard" we are settling on. Any manual change will currently automatically reverted.

As I learned to get used to that the formatting is kind of part of the language syntax, and now even believe it's a good thing as it all saves us from fruitless discussions, I think we should just take over what goimport does for us.

Tbh, as I'm using Intellij IDEA which always hides the imports (as it does for packages in Java) I even don't see this import list by default.

I think we have bigger fish to fry.

Footnote: In my 35+ years of coding I went through various stages of zealoticy when it comes to code formatting. I was never super dogmatic, but I did care. And then I stopped to care and saved the energy for something else. I only regret that I didn't have done this earlier. I really find it so refreshing that golang included the coding style as part of the language. Not everyone likes this, but it immediately stopped any discussion. And one quickly gets over it, believe me :)

I wonder why this is not the case for imports (not sure here). But I'm all for letting this done automatically by a tool like goimports and stop worrying (but get accustomized to it. It only hurts for a short time, believe me ;-)
